README.md

```diff
@@ -67,3 +71,7 @@ Table of commands:
 | pdm run bootstrap | Install pre-commit hooks                           |
 | pdm run test      | Run Python tests                                   |
 | pdm run coverage  | Get code coverage data and open results in browser |
+
+## Sentry staff info
+
+Google Cloud Build will automatically build a Docker image when the code merges on `master`. Logging to Google Cloud Run and deploy the latest image.
```

Comment:
I have this working for `gitbot`, however, I did not manage to make it work. This is just documenting it for us to know what to do.